### PR TITLE
wire it all up

### PR DESCRIPTION
**Pull request by saibotsivad**
woooooooot

### CODE REVIEW

File: client/state/workbook-and-sheet-module.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+
+verbs:
+
+- load a workbook from google (requires workbook id)
+- load a sheet from google (requires workbook and sheet id)
```
Comment (TehShrike) [inline client/state/workbook-and-sheet-module.js]: unnecessary!!!111ynarst

**Review by TehShrike — CHANGES_REQUESTED** [review]

File: client/state/workbook-and-sheet-module.js (hunk 1 above)
Comment (saibotsivad) [reply client/state/workbook-and-sheet-module.js]: 😡

Comment (TehShrike) [reply client/state/workbook-and-sheet-module.js]: 🐳

Comment (saibotsivad) [reply client/state/workbook-and-sheet-module.js]: ❤️

File: client/state/workbook-and-sheet-module.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+			state.sheet = sheet
+		}
+	},
+	actions: {
+		async fetchWorkbook(context, workbookId) {
+			const workbook = await getWorkbook(workbookId, httpGet)
+			context.commit('setWorkbook', workbook)
+		},
+		async fetchSheet(context, sheetId) {
+			const sheet = await getSheet(sheetId, httpGet)
```
Comment (TehShrike) [inline client/state/workbook-and-sheet-module.js]: This needs to be passed the workbookId as well

**Review by TehShrike — CHANGES_REQUESTED** [review]

**Review by TehShrike — APPROVED** [review]
lgtm

**Pull request merged**